### PR TITLE
Fixes incorrect return methods in PHPDocs

### PR DESCRIPTION
**Pull request by NiklasBr**
## Changes in this pull request  
Resolves Documentation vs code mismatches

### CODE REVIEW

File: models/DataObject/Listing.php [hunk 1]
```diff
@@ -178,7 +178,7 @@ public function count()
      * @param int $offset
      * @param int $itemCountPerPage
      *
-     * @return array
+     * @return int
```
Comment (blankse) [inline models/DataObject/Listing.php]: I think `@return array` was correct

Comment (brusch) [reply models/DataObject/Listing.php]: Yep, count should return `int`

Comment (NiklasBr) [reply models/DataObject/Listing.php]: In that `case @method int load()` should be `@method array load()`.

I assumed the higher order one was the correct one.

Comment (NiklasBr) [reply models/DataObject/Listing.php]: (line 26 by the way)

Comment (blankse) [reply models/DataObject/Listing.php]: In Line 26 is:
https://github.com/pimcore/pimcore/blob/50c71ce7ff1cc75caff5b427b8a09cdb373b434b/models/DataObject/Listing.php#L26

You can change it here also to `@return Model\DataObject[]`

File: models/DataObject/Listing.php [hunk 2]
```diff
@@ -228,7 +228,7 @@ public function key()
     }
 
     /**
-     * @return mixed|void
+     * @return mixed|null
```
Comment (blankse) [inline models/DataObject/Listing.php]: I think `@return mixed` should be enough. `null` is impossible. The `next` function will return `false` if the array is at the end.

**Pull request merged**